Cargo.toml

```diff
@@ -19,4 +19,8 @@ resolver = "2"
 members = [
     "proxy",
     "tag-generator",
+    "tasks",
 ]
+
+[patch.crates-io]
+imap-types = { git = "https://github.com/duesee/imap-codec" }
```

Comment:
This makes it simpler for now to work in lock-step with `imap-codec`. Should I file an issue as a reminder to remove this?